gpu-operator-must-gather.sh

```
@@ -0,0 +1,256 @@
+    $K get clusterversion/version -oyaml > $ARTIFACT_DIR/openshift_version.yaml
+fi
+
+echo "Get the operator namespaces"
+OPERATOR_POD_NAME=$($K get pods -lapp=gpu-operator -oname -A)
+
+if [ -z "$OPERATOR_POD_NAME" ]; then
+    echo "FATAL: could not find the GPU Operator Pod ..."
+    exit 1
+fi
+
+OPERATOR_NAMESPACE=$($K get pods -lapp=gpu-operator -A -ojsonpath={.items[].metadata.namespace} --ignore-not-found)
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)